.nycrc.yml

```diff
@@ -0,0 +1,2 @@
+# https://github.com/istanbuljs/nyc#common-configuration-options
+all: true
```

Comment:
The coverage report file (`lcov.info`) is empty if we do not specify `all : true`.